### ISSUE TITLE
[BUG] syntax error at or near "LIMIT" for Common Table Expressions

### ISSUE THREAD

**Issue by mirkadev** [cweijan/vscode-mysql#1140]
- Database type and version:
Postgres 16

There is an error `syntax error at or near "LIMIT"` for the next request:

```sql
WITH duplicate_tables AS (
    SELECT
        id,
        started_at,
        finished_at,
        company_name,
        ROW_NUMBER() OVER(PARTITION BY started_at, finished_at, company_name) AS row_num
    FROM experiences
    WHERE deleted_at is null
)
delete from table
where id in (select id from duplicate_tables);
```

But in another client it's ok. I guess it's common problem for all databases and `with` request

**Comment by cweijan** [comment]
Thanks for the feedback. This is caused by automatic paging. You can temporarily turn it off and wait for my fix in future versions.

![image](https://github.com/cweijan/vscode-database-client/assets/27798227/028f32e2-01c3-4aef-8341-fa9c84fb12dc)

**Comment by mirkadev** [comment]
Thanks

**Comment by cweijan** [comment]
Fixed in version 7.3.8.

**Comment by mirkadev** [comment]
Thank you, it works
